src/app/content/highlights/components/CardWrapper.tsx

```diff
@@ -29,12 +31,47 @@ const Wrapper = ({highlights, className, container, highlighter}: WrapperProps)
   const [cardsPositions, setCardsPositions] = React.useState<Map<string, number>>(new Map());
   const [cardsHeights, setCardsHeights] = React.useState<Map<string, number>>(new Map());
   const [offsets, setOffsets] = React.useState<Map<string, { top: number, bottom: number }>>(new Map());
+  const [shouldFocusCard, setShouldFocusCard] = React.useState(false);
   const focusedId = useSelector(focused);
   const focusedHighlight = React.useMemo(
     () => highlights.find((highlight) => highlight.id === focusedId),
     [focusedId, highlights]);
   const prevFocusedHighlightId = React.useRef(focusedId);
 
+  // This function is triggered by keyboard shortuct defined in useKeyCombination(...)
+  // If user used it while having focus outside of this component and when focusedHighlight is truthy
+  // then we want move focus to the proper Card component by setting shouldFocusCard to true.
+  // If user used it while having focus inside of this component then we want to
+  // focus highlight in the content for the focusedHighlight and reset shouldFocusCard to false.
+  const moveFocus = React.useCallback(() => {
+    const document = assertDocument();
+    const activeElement = document.activeElement;
+
+    if (!focusedHighlight || !activeElement || !element.current) {
+      return;
+    }
+
+    if (element.current.contains(activeElement)) {
+      let highlightElement: HTMLElement | undefined | null;
+      focusedHighlight.elements.some((el) => {
+        highlightElement = (el as HTMLElement).querySelector('[data-for-screenreaders]') as HTMLElement | null;
+        return highlightElement;
+      });
+      if (highlightElement) {
+        highlightElement.focus();
```

Comment:
yea this logic is gnarly, i'd rather use highlighter.focus() and make that focus the proper element

Reply:
Do you mean `highlighter.focus()` or `highlight.focus()`?
We have `highlight.focus()` but it is used in `highlightManager.update()` and it is called multiple times when interacting with highlights' cards due to rerenders, so we won't be able to move focus in this method unless we want to work on `highlightManager` to debug this

On the other hand, I could add `focus` method on `highlighter` that does what you say - I'm just confused with naming, because we already have `highlighter.clearFocus` and `highligh.focus` methods that are removing/adding `FOCUS_CSS` classes to the elements.

Reply:
or we could add a method on a highlight that will return it's screen reader nodes, sth like
`highlight.getScreenReaderNodes(): [startNode, endNode]` and we could use it like `highlight.getScreenReaderNodes()[0].focus`

Reply:
One more idea would be to change current name from `focus` to `applyFocusedStyles` and then add `focus` method that actually focuses the highlight